pipeline/buildpacks/0.1/README.md

```diff
@@ -0,0 +1,89 @@
+# Buildpacks
+
+This pipeline builds source into a container image using [Cloud Native Buildpacks](https://buildpacks.io). To do that, it uses [builders](https://buildpacks.io/docs/concepts/components/builder/#what-is-a-builder) to run buildpacks against your application source.
+
+> _**What are Cloud Native Buildpacks?**_
+> 
+> _Cloud Native Buildpacks are pluggable, modular tools that transform application source code into OCI images. They replace Dockerfiles in the app development lifecycle, and enable for swift rebasing of images and modular control over images (through the use of builders), among other benefits._
+
+## Dependencies
+
+- [`git-clone` task](https://github.com/tektoncd/catalog/tree/master/task/git-clone) 0.2 or newer
+- [`buildpacks` task](https://github.com/tektoncd/catalog/tree/master/task/buildpacks/) 0.3 or newer
+- [`buildpacks-phases` task](https://github.com/tektoncd/catalog/tree/master/task/buildpacks-phases/) 0.2 or newer
+
+## Compatibility
+
+- **Tekton** v0.17.0 and above
+- **[Platform API][platform-api]** 0.4
+
+[platform-api]: https://buildpacks.io/docs/reference/spec/platform-api/
+
+## Install
```

Comment:
Can you please also add the command to install `git-clone` task ?

Reply:
I certainly can but felt hesitant due to this [comment](https://github.com/tektoncd/pipeline/discussions/3822#discussioncomment-410572):

> The `git-clone` task that one org uses might be different to the `git-clone` task another org uses - for example, one might use a private image that has been approved by their security team or may be holding back to an older version. For that reason I think it's best to declare the dependency in the README and let the user decide to use their own or install from the catalog.

I suppose I can add a secondary block for dependencies. For example:

--- 

#### Install dependencies (if missing):

```
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.2/git-clone.yaml
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/buildpacks/0.3/buildpacks.yaml
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/buildpacks-phases/0.2/buildpacks-phases.yaml
```

#### Install pipeline:

```
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/pipeline/buildpacks/0.1/buildpacks.yaml
```